### PR TITLE
Relay support for Azure Service Bus

### PR DESCRIPTION
**Pull request by blinktag**

### CODE REVIEW

File: backends/azure/relay.go [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+		return nil
+	}
+
+	if r.Queue != nil {
+		return r.readQueue(ctx, handler)
+	}
+
+	return r.readTopic(ctx, handler)
+
+	return nil
```
Comment (dselans) [inline backends/azure/relay.go:159]: Extra return

File: relay/azure.go [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+	}
+
+	azureRecord := convertAzureMessageToProtobufRecord(msg.Value)
+
+	client := services.NewGRPCCollectorClient(conn)
+
+	if _, err := client.AddAzureRecord(ctx, &services.AzureRecordRequest{
+		Records: []*records.AzureSinkRecord{azureRecord},
+	}); err != nil {
+		fmt.Printf("%+v", azureRecord)
```
Comment (dselans) [inline relay/azure.go:29]: Is this debug?

File: relay/relay.go [hunk 3]
```diff
@@ -170,9 +171,12 @@ func (r *Relay) Run(id int, conn *grpc.ClientConn, ctx context.Context) {
 		case *rabbitTypes.RelayMessage:
 			r.log.Debugf("Run() received rabbit message %+v", v)
 			err = r.handleRabbit(ctx, conn, v)
-		case *kafkatypes.RelayMessage:
+		case *kafkaTypes.RelayMessage:
```
Comment (dselans) [inline relay/relay.go:174]: ❤️

**Review by dselans — APPROVED** [review]
Couple of minor fixes. Looks great!

File: backends/azure/relay.go (hunk 1 above)
Comment (blinktag) [reply backends/azure/relay.go:159]: Fixed!

File: relay/azure.go (hunk 2 above)
Comment (blinktag) [reply relay/azure.go:29]: Yes, fixed!

**Pull request merged**